ultravox/data/datasets.py

```diff
@@ -71,8 +72,9 @@ class DataCollatorForSeq2SeqWithAudio(transformers.DataCollatorForSeq2Seq):
     def __call__(self, features, *args, **kwargs):
         audio_features = [f.pop("audio_values") for f in features]
         batch = super().__call__(features, *args, **kwargs)
-        batch["audio_values"] = torch.nn.utils.rnn.pad_sequence(
-            audio_features, batch_first=True
+        max_len = max([x.shape[-1] for x in audio_features])
```

Comment:
Please add a comment explaining what's going on here.

Reply:
Done. Here's the comment for reference:
> Pad the last dimension of all audio_values to the same length, with 0s on the right.